paasta_tools/utils.py

```diff
@@ -997,8 +998,24 @@ def get_net(self) -> str:
         """
         return self.config_dict.get("net", "bridge")
 
+    def has_bulkdata(
+        self,
+    ) -> bool:
+        return self.config_dict.get("uses_bulkdata", True)
+
     def get_volumes(self, system_volumes: Sequence[DockerVolume]) -> List[DockerVolume]:
         volumes = list(system_volumes) + list(self.get_extra_volumes())
+        # add a dictionary to docker_volumes with a hostPath of /nail/bulkdata
+        # if the service uses bulkdata
+        if self.has_bulkdata():
+            # the volume has a mode of RO
+            volumes = volumes + [
+                {
+                    "hostPath": "/nail/bulkdata",
+                    "containerPath": "/nail/bulkdata",
+                    "mode": "RO",
+                }
+            ]
```

Comment:
nit:
```suggestion
            volumes.append(
                {
                    "hostPath": "/nail/bulkdata",
                    "containerPath": "/nail/bulkdata",
                    "mode": "RO",
                }
            )
```
is slightly preferable since it won't cause an entirely new list to be created (`append` will resize the existing list, whereas `+` results in a new list being created)

e.g.:
```
>>> test = []
>>> id(test)
139872597201776
>>> test = test + [{1:1}]
>>> id(test)
139872597231328
```
vs
```
>>> test = []
>>> id(test)
139872597201776
>>> test.append({1:1})
>>> id(test)
139872597201776
```